Xamarin.Forms.Core/FileMediaSource.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+	[TypeConverter(typeof(FileImageSourceConverter))]
```

Comment:
```suggestion
	[TypeConverter(typeof(FileMediaSourceConverter))]
```
Also need to create class `FileMediaSourceConverter`